omod/src/main/java/org/openmrs/module/dhisreport/web/controller/ReportController.java

```diff
@@ -444,8 +444,15 @@ AdxType getAdxType( DataValueSet dvs, String timeperiod )
         {
             DataValueType dvtype = new DataValueType();
             dvtype.setDataElement( dv.getDataElement() );
-            dvtype.setValue( new BigDecimal( dv.getValue() ) );
+            try
+            {
+            BigDecimal bd = new BigDecimal( dv.getValue() );
+            dvtype.setValue( bd );
             dvTypeList.add( dvtype );
+            }
+            catch ( NumberFormatException e )
+            {
```

Comment:
It is not recommended to catch exceptions and do nothing. You would rather leave them to bubble through.

Reply:
You seem to have nothing to do in the exception handler. Can't you just remove the try and catch to let it bubble up?